robotoff/utils/text.py

```diff
@@ -47,3 +49,12 @@ def get_nlp(lang: str):
 FR_NLP_CACHE = cache.CachedStore(
     functools.partial(get_nlp, lang="fr"), expiration_interval=None
 )
+
+
+def get_fr_known_tokens() -> Set[str]:
```

Comment:
This function is quite specific to ingredient spellcheck (we load the list of ingredient token on the first line). I think it would be more appropriate if it is in the spellcheck module.

Reply:
Code moved to module in https://github.com/openfoodfacts/robotoff/pull/181/commits/dc2a3cca51b8bce384f2f86a9b89f4000ee9a2e5